services/QuillLMS/client/app/bundles/Diagnostic/components/incorrectSequence/incorrectSequenceContainer.jsx

```diff
@@ -29,28 +32,16 @@ class IncorrectSequencesContainer extends Component {
     };
   }
 
-  UNSAFE_componentWillMount() {
-    const { actionFile } = this.state;
-    const { dispatch, match } = this.props;
-    const { params } = match;
-    const { questionID } = params;
-    const type = window.location.href.includes('sentence-fragments') ? 'sentence-fragment' : 'sentence-combining'
-    dispatch(actionFile.getUsedSequences(questionID, type))
-  }
-
-  componentDidUpdate(previousProps, previousState) {
-    const question = this.props[this.state.questionType].data[this.props.match.params.questionID]
-    const incorrectSequences = this.getSequences(question)
-
-    if (_.isEqual(previousState.incorrectSequences, incorrectSequences)) return;
-
-    this.setState({ incorrectSequences, orderedIds: null, });
-  }
-
-  getSequenceIndexByKey = (key) => {
-    const { incorrectSequences, } = this.state
+  UNSAFE_componentWillReceiveProps(nextProps) {
```

Comment:
Ideally this would be rewritten as well -- I know we already had an `UNSAFE_` function in this file but if you're already doing the work of adding new functionality and having to QA it and all that, it's a good time to clean up stuff like this.